sp/src/game/server/ez2/npc_flyingpredator.cpp

```diff
@@ -639,6 +896,28 @@ void CNPC_FlyingPredator::OnFed()
 	}
 }
 
+//-----------------------------------------------------------------------------
+// Purpose: 
+// Input  : &info - 
+// Output : Returns true on success, false on failure.
+//-----------------------------------------------------------------------------
+bool CNPC_FlyingPredator::ShouldGib( const CTakeDamageInfo &info )
+{
+	// If the damage type is "always gib", we better gib!
+	if ( info.GetDamageType() & DMG_ALWAYSGIB )
+		return true;
+
+	// Stukapups always gib
+	if ( m_bIsBaby )
+		return true;
+
+	// Stukabats gib for any non-crush, non-blast damage greater than 15 (TODO: cvar?)
+	if ( info.GetDamage() > 15.0f || (info.GetDamageType() & (DMG_CRUSH | DMG_BLAST) ) )
```

Comment:
I want to revisit this later. Perhaps we can implement the exploding ragdolls concept from way back